client/src/components/Dataset/DatasetStorage/DatasetStorage.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <div>
+        <h3 v-if="includeTitle">Dataset Storage</h3>
+        <loading-span v-if="storageInfo == null"> </loading-span>
+        <div v-else>
+            <p v-if="storageInfo.name">
+                This data is stored in a Galaxy ObjectStore named <b>{{ storageInfo.name }}</b
+                >.
+            </p>
+            <p v-if="storageInfo.object_store_id">
```

Comment:
If both name and id are defined this seems to take up more space than needed, maybe you could combine that into one sentence (or a small table?)?

Reply:
This was mean t to be an else-if I think. Thanks for the catch!